statsmodels/tsa/api.py

```diff
@@ -24,3 +24,4 @@
 from .statespace.dynamic_factor import DynamicFactor
 from .regime_switching.markov_regression import MarkovRegression
 from .regime_switching.markov_autoregression import MarkovAutoregression
+from .holtwinters import holt_winters, holt, simple_exp_smoothing
```

Comment:
Should this module be called exponential_smoothing to enhance discoverability?

Reply:
Called the class ExponentialSmoothing